ui/HomePage.py

```
@@ -0,0 +1,35 @@
+import streamlit as st
+import pandas as pd
+import numpy as np
```

Ruff

`numpy` imported but unused

Remove unused import: `numpy`

(F401)